administrator/language/de-DE/de-DE.plg_captcha_recaptcha.ini

```diff
@@ -4,76 +4,73 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
 ; Note : All ini files need to be saved as UTF-8
 
-PLG_CAPTCHA_RECAPTCHA_XML_DESCRIPTION="Dieses Captcha-Plugin verwendet den Google reCAPTCHA-Service um Spammer fernzuhalten. Google bietet diesen kostenlosen Service auf der Website <a href="_QQ_"https://www.google.com/recaptcha"_QQ_" target="_QQ_"_blank"_QQ_" rel="_QQ_"noopener noreferrer"_QQ_">https://www.google.com/recaptcha</a> an. Dort sind ein Websiteschlüssel und ein geheimer Schlüssel für die eigene Website erhältlich. Um diese Funktion für neue Benutzerregistrierungen nutzen zu können, den Capthcha-Dienst „Captcha - reCAPTCHA“ in den Optionen unter dem Menüpunkt „Benutzer“ auswählen."
-PLG_CAPTCHA_RECAPTCHA="Captcha - ReCAPTCHA"
 
+PLG_CAPTCHA_RECAPTCHA="Captcha - ReCAPTCHA"
+PLG_CAPTCHA_RECAPTCHA_XML_DESCRIPTION="Dieses Captcha-Plugin verwendet den Google reCAPTCHA-Service um Spammer fernzuhalten. Google bietet diesen kostenlosen Service auf der Website <a href="_QQ_"https://www.google.com/recaptcha"_QQ_" target="_QQ_"_blank"_QQ_" rel="_QQ_"noopener noreferrer"_QQ_">https://www.google.com/recaptcha</a> an. Dort sind ein Websiteschlüssel und ein geheimer Schlüssel für die eigene Website erhältlich. Um diese Funktion für neue Benutzerregistrierungen nutzen zu können, den Capthcha-Dienst „Captcha - reCAPTCHA“ in den Optionen unter dem Menüpunkt „Benutzer“ auswählen."
```

Comment:
typo and wording

Dieses Captcha-Plugin verwendet den Google reCAPTCHA-Service um Spammer fernzuhalten. Google bietet diesen kostenlosen Service auf der Website <a href="_QQ_"https://www.google.com/recaptcha"_QQ_" target="_QQ_"_blank"_QQ_" rel="_QQ_"noopener noreferrer"_QQ_">https://www.google.com/recaptcha</a> an. Dort sind ein Websiteschlüssel und ein geheimer Schlüssel für die eigene Webseite erhältlich. Um diese Funktion für neue Benutzerregistrierungen nutzen zu können, muss der Captcha-Dienst „Captcha - reCAPTCHA“ in den Optionen unter dem Menüpunkt „Benutzer“ ausgewählt sein.